config/kubernetes/500-webhook.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2024 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: manual-approval-gate-webhook
+  namespace: tekton-pipelines
+  labels:
+    version: "devel"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: manual-approval
```

Comment:
IMO, selector label looks too generic.
Select a label very specific to `manual-approval-gate` pods